src/test/integration/controllers/accounts.controller.e2e-spec.ts

```diff
@@ -0,0 +1,252 @@
+                count: 57,
+            },
+        ].forEach(({ ownerAddress, count }) => {
+            describe(`ownerAddress = ${ownerAddress}`, () => {
+                it(`should return count of accounts searched by owner address ${ownerAddress}`, async () => {
+                    const params = new URLSearchParams({
+                        'ownerAddress': `${ownerAddress}`,
+                    });
+
+                    await request(app.getHttpServer())
+                        .get(`${path}/count?${params}`)
+                        .expect(200)
+                        .then(res => {
+                            expect(+res.text).toBeGreaterThanOrEqual(count);
+                        });
+                });
+            });
+        });
+    });
+
+    describe('/accounts/{address}', () => {
+        it('should return account details for a given address', async () => {
+            const address: string = 'erd1qqqqqqqqqqqqqpgqksuhk8dal6wyqje72csqma5j6ncguwj4zvdqc6xzwj';
+
+            await request(app.getHttpServer())
+                .get(`${path}/${address}`)
+                .expect(200)
+                .then(res => {
+                    expect(res.body.address).toStrictEqual(address);
+                    expect(res.body.balance).toStrictEqual('333118000000000000000');
+                    expect(res.body.nonce).toStrictEqual(0);
+                    expect(res.body.timestamp).toStrictEqual(1700842188);
+                    expect(res.body.shard).toStrictEqual(2);
+                    expect(res.body.ownerAddress).toStrictEqual('erd1ajyez9tt0a9sra5mvm44g0rlzzg9yfytrw3twt0z8dr4wg7zmlwq7w8xv6');
+                    expect(res.body.assets).toBeDefined();
+                    expect(res.body.code).toBeDefined();
+                    expect(res.body.codeHash).toStrictEqual('KiHTQKSCnk9+wjZZL0jeS2vV2E3dIVxrG0CgNqoroJc=');
+                    expect(res.body.rootHash).toStrictEqual('at+/lZ/9vwrRPDLkwXau559k7n8qyJ0DVm8tMIgcBuU=');
+                    expect(res.body.txCount).toStrictEqual(38364);
+                    expect(res.body.scrCount).toStrictEqual(32687);
+                    expect(res.body.developerReward).toStrictEqual('902351461371000000');
+                    expect(res.body.isPayableBySmartContract).toStrictEqual(true);
+                    expect(res.body.isUpgradeable).toStrictEqual(true);
+                    expect(res.body.isReadable).toStrictEqual(true);
+                    expect(res.body.isGuarded).toStrictEqual(false);
+                    expect(res.body.isPayable).toStrictEqual(false);
+                    expect(res.body.deployTxHash).toStrictEqual('215e24332cc2734e499be534268a7badd13dcf82735cfc0ab571fe09d9fe0804');
+                    expect(res.body.deployedAt).toStrictEqual(1689340068);
+                });
+        });
+    });
+
+    describe('Validations', () => {
+        it('should return 400 Bad Request for an invalid address', async () => {
```

Comment:
This test is the same with the following test:
- should return 400 Bad Request if owner address is not correct